scripts/ingest-tarball.sh

```diff
@@ -182,7 +182,7 @@ function ingest_compat_tarball() {
         echo_yellow "Removing the existing layer, and adding the new one from the tarball..."
         cvmfs_server transaction "${repo}"
         rm -rf "/cvmfs/${repo}/${basedir}/${version}/compat/${os}/${arch}/"
```

Comment:
@bedroge We should probably check with CVMFS devs if there's a recommend approach for this type of "replace" operation...

Ideally we don't need to unpack the tarball at all, without having to publish the removal first

Reply:
That does require a change in the tarball itself, as we can't simply ingest it to the `versions` base dir. We would either need to modify the way the tarball is created, or somehow strip the first directory (`2021.12`) from every file/dir in the tarball and replace it by / ingest it to `2021.12-<n>`.

Reply:
Since we strongly prefer using `cmvfs_server` ingest, we should change the way the tarball is created.

The ingest script could then also fail hard if the top-level directory isn't what's expected (`2021.12.<n+1>` if the current is `2021.12.<n>`).

I would prefer using a versioning scheme like `2021.12-001`, `2021.12-002` (since that sorts better)